discovery-infra/start_discovery.py

```diff
@@ -229,12 +230,25 @@ def set_cluster_vips(client, cluster_id, machine_net):
     client.update_cluster(cluster_id, cluster_info)
 
 
-def set_cluster_machine_cidr(client, cluster_id, machine_net):
+def set_cluster_machine_cidr(client, cluster_id, machine_net, set_vip_dhcp_allocation=True):
     cluster_info = client.cluster_get(cluster_id)
-    cluster_info.vip_dhcp_allocation = True
-    cluster_info.machine_network_cidr = machine_net.cidr_v4
+    if set_vip_dhcp_allocation:
```

Comment:
no need for if ,just 
cluster_info.vip_dhcp_allocation = set_vip_dhcp_allocation

Reply:
OK

Reply:
Actually since python is typeless,  prefer to set it to True explicitly